src/main/java/core/basesyntax/Calculator.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Calculator {
+    public double calculator(double number1, double number2, char operating) {
+        switch (operating) {
+            case '+':
+                return number1 + number2;
+            case '-':
+                return number1 - number2;
+            case '/':
+                return number1 / number2;
+            case '*':
+                return number1 * number2;
+            case '^':
+                if (number1 < 0) {
+                    return Math.pow(number1 * -1, number2) * -1;
```

Comment:
```suggestion
                    return Math.pow(number1 * -1, number2);
```
does Math as a science allow such manipulations? P.S.: better to throw exception here

Reply:
Mathematics allows this, it seems to me better to leave it like this

Reply:
and why have you done this in the first place?